### DISCUSSION TITLE
fastapi doesn't use host passed to uvicorn

### DISCUSSION THREAD

**Discussion by electronhead** [fastapi/fastapi#6631] [Questions]
FastApi output log shows a host different from the one supplied to uvicorn. 

Two computers, both running fastapi. I log into computer #1 through an SSH tunnel and run fastapi using host-x. I log into computer #1 again and then ssh into computer #2 and run fastapi there using a different host-y. I then send requests to the different hosts, each of which returns 200 OK.

The problem is that fastapi running on computer #2 uses the host corresponding to the computer #1 host even though uvicorn is supplied a different host. If I add a third computer logged into from computer #1, the same thing happens.

One difference is that the host on computer#1 is shown in the eth0 section of ifconfig whereas the host on computer #2 is shown in the wlan0 section of ifconfig. So maybe fastapi is somehow resolving the wlan0 host of computer #2 to the eth0 host of computer #1 on the other side of the ssh connection.

Ultimately, how can I force fastapi to use the same host as uvicorn? Where is fastapi picking up computer #1's host? Is this purely a networking problem, or can fastapi take a parameter (that I haven't found) that makes sure that uvicorn and fastapi agree on the host? The code currently initializes the FastAPI object with no arguments.

### running on computer #1 (logged into through ssh tunnel)
(venv) gary@batterypi:~/thenthis $ python run.py --host **192.168.1.94**
INFO:     Started server process [23966]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://**192.168.1.94**:8000 (Press CTRL+C to quit)
INFO:     **192.168.1.94**:46096 - "GET /actions/system_info/execute HTTP/1.1" 200 OK

### running on computer #2 (logged into from computer #1)
(venv) gary@remotepi:~/remoteapi $ python run.py --host **192.168.1.214**
INFO:     Started server process [27012]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://**192.168.1.214**:8000 (Press CTRL+C to quit)
INFO:     **192.168.1.94**:58604 - "GET /actions/system_info/execute HTTP/1.1" 200 OK

[run.py.pdf](https://github.com/tiangolo/fastapi/files/6454989/run.py.pdf)

**Comment by tiangolo** [comment]
I assume you solved it, thanks for closing the issue 👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
